src/management/fs.c

```diff
@@ -187,6 +191,35 @@ static int fs_create_process_data(struct controller_api *fs_create)
 					       JSON_C_TO_STRING_SPACED);
 
 		fs_create_api->req.fs_options = fs_options;
+
+		json_fs_bsize_obj = json_tokener_parse(fs_options);
+		json_object_object_get_ex(json_fs_bsize_obj, "fs_bsize",
+					  &json_fs_bsize_obj1);
+		if (json_fs_bsize_obj1 != NULL) {
+			str = json_object_get_string(json_fs_bsize_obj1);
+			if (str) {
+				fs_bsize = (size_t) strtol(str, (char **)NULL,
+					   10);
+				if (fs_bsize < CFS_MIN_BLOCKSIZE ||
+				    fs_bsize > CFS_MAX_BLOCKSIZE) {
+					log_info("fs bsize range invalid: %lu",
```

Comment:
If the block size is invalid, should _cfs_fs_create_ fail instead of continuing with default size?